portability-transfer/src/main/java/org/datatransferproject/transfer/logging/EncryptingLayout.java

```diff
@@ -0,0 +1,32 @@
+package org.datatransferproject.transfer.logging;
+
+import java.util.UUID;
+import org.apache.log4j.Layout;
+import org.apache.log4j.helpers.ISO8601DateFormat;
+import org.apache.log4j.spi.LoggingEvent;
+
+public class EncryptingLayout extends Layout {
+
+  private final UUID jobId;
+
+  public EncryptingLayout(UUID jobId) {
+    this.jobId = jobId;
+  }
+
+  @Override
+  public String format(LoggingEvent event) {
+    // NB: copied from SimpleLayout.format()
+    return String.format("[%s] [%s]: %s - %s%s", new ISO8601DateFormat().format(event.timeStamp),
```

Comment:
Can this be a pattern that is passed in instead of hard coded?

Reply:
Do you mean different kinds of patterns for EncryptingLayout, or different kinds of patterns for different layouts?

Reply:
I mean this kind of string seems like it should come from the log4j.properties file, but that be left as a TODO

Reply:
It can't be in the log4j properties file, since that pattern layout doesn't have room for arbitrary variables.  But it can go somewhere else.